tap_stripe/__init__.py

```diff
@@ -1093,6 +1135,13 @@ def main():
         print(json.dumps(catalog, indent=2))
     # Otherwise run in sync mode
     else:
+        Context.window_size = get_date_window_size('date_window_size', DEFAULT_DATE_WINDOW_SIZE)
+        Context.event_window_size = get_date_window_size('event_date_window_size', DEFAULT_EVENT_DATE_WINDOW_SIZE)
+        # Reset event_window_size to 30 days if it is greater than 30 because Stripe Event API returns data of the last 30 days only.
+        if Context.event_window_size > 30:
```

Comment:
Add Unit test of this.

Reply:
If not possible add Integration tests

Reply:
Yes, We have kept value of `event_window_size` 35(`> 30`) to cover in tap-tester.